gormsupport/lifecycle.go

```diff
@@ -10,9 +10,9 @@ import (
 // The Lifecycle struct contains all the items from gorm.Model except the ID field,
 // hence we can embed the Lifecycle struct into Models that needs soft delete and alike.
 type Lifecycle struct {
-	CreatedAt time.Time
-	UpdatedAt time.Time
-	DeletedAt *time.Time
+	CreatedAt time.Time  `yaml:"created_at"`
```

Comment:
No, I don't need them. You're right. Fixed in ef18a30e9bffcb293fc7f2ff18fb8f4f5f37de65. Those are probably left-overs from my early days of pressing work item types into YAML. Thanks for pointing this out.